### DISCUSSION TITLE
How do you send a message to yourself?

### DISCUSSION THREAD

**Discussion by josecyc** [tulir/whatsmeow#388] [WhatsApp protocol Q&A]
Everything else is working fine except sending a message to myself I can't get it to work. Any pointers on this?

**Comment by arthurbaquit** [comment]
Apparently it was answered [here](https://github.com/tulir/whatsmeow/discussions/312)

> just save your contact number with any name , and open contacts which you have saved your name in the whatsapp , then you can able to message yourself

**Comment by alvaroeds** [comment]
Puede suar


	jid := types.NewJID("your-number", "s.whatsapp.net")
	resp, err := client.SendMessage(context.Background(), jid, client.BuildPollCreation("miau?", []string{"sí", "no"}, 1))
	if err != nil {
		panic(err)
	}
	fmt.Println("Send message:", resp)

Para enviarse un mensaje a si mismo

**Comment by roniahmad** [comment]
it is the same as you send to other number.
if you check whether the message is from you your self in your event handler, you should comment that code.



`if evt.Info.IsFromMe{
}`
